tests/cupy_tests/core_tests/test_ndarray.py

```diff
@@ -217,7 +218,7 @@ def test_cuda_array_interface(self):
         assert not iface['data'][1]
         assert iface['descr'] == [('', '<f8')]
         assert iface['strides'] is None
-        assert iface['stream'] == 1
+        assert iface['stream'] == 2 if stream_module.is_ptds_enabled() else 1
```

Comment:
using the above proposed function cleans this code

Reply:
Done in 7a938a8.